lib/provisioner/plugin/automator.rb

```diff
@@ -66,6 +68,20 @@ def runTask
         end
       end
 
+      def verify_ssh_host_key(host, type = 'rsa')
```

Comment:
The `ssh_keyscan` utility function is reusable. It has a default type of `rsa` if nothing is set. The `verify_ssh_host_key` is a function that consumes `ssh_keyscan` and while it is not 100% required that it specify the type, if the underlying method changed, it would affect this method.